packages/run-protocol/src/proposals/core-proposal.js

```diff
@@ -193,6 +193,8 @@ const RUN_STAKE_MANIFEST = harden({
   },
   [econBehaviors.startRunStake.name]: {
     consume: {
+      board: 'board',
+      chainStorage: true,
```

Comment:
good... I wonder if test coverage prompted this or if you just remembered.

Reply:
I don't recall.